src/dafka_consumer.c

```diff
@@ -134,11 +139,13 @@ s_subscribe (dafka_consumer_t *self, const char *topic)
     dafka_proto_subscribe (self->consumer_sub, DAFKA_PROTO_MSG, topic);
     dafka_proto_subscribe (self->consumer_sub, DAFKA_PROTO_HEAD, topic);
 
-    if (self->verbose)
-        zsys_debug ("Consumer: Send EARLIEST message for topic %s", topic);
+    if (!self->reset_latest) {
+        if (self->verbose)
+            zsys_debug ("Consumer: Send EARLIEST message for topic %s", topic);
```

Comment:
You only send earlies on reset_latest?
Why not on earliest?

Reply:
Notice the exclamation mark :)

I know the variables names are not that nice, will improve on that if I got some free time

Reply:
got it. make sense.
It is also make sense to change the message name to LATEST. because it is actually give you the latest heads on all topics. maybe even GET_HEADS

Reply:
Yes, I like GET_HEADS